.storybook/preview.js

```diff
@@ -0,0 +1,15 @@
+import { action } from "@storybook/addon-actions";
+import { configure } from "@storybook/react";
+
+import "./stories.css";
+
+configure(require.context("../src", true, /\.stories\.mdx$/), module);
+
+global.___loader = {
+  enqueue: () => {},
+  hovering: () => {},
+};
+global.__PATH_PREFIX__ = "";
+window.___navigate = (pathname) => {
+  action("NavigateTo:")(pathname);
+};
```

Comment:
these all seem to be dummy implementations? do the docs suggest to do it this way or can we safely remove the noise?

Reply:
You're right ! I removed it and more.